superagi/vector_store/vector_factory.py

```diff
@@ -61,4 +64,18 @@ def get_vector_storage(cls, vector_store: VectorStoreType, index_name, embedding
             )
             return weaviate.Weaviate(client, embedding_model, index_name, 'text')
 
+        if vector_store == VectorStoreType.QDRANT:
```

Comment:
add unit test for vector factory

Reply:
https://github.com/TransformerOptimus/SuperAGI/pull/742